MekHQ/src/mekhq/gui/PlanetarySystemMapPanel.java

```diff
@@ -393,10 +394,10 @@ protected void paintComponent(Graphics g) {
     private void chooseFont(Graphics g, PlanetarySystem system, Campaign c, int limit) {
         // start with 16
         int fontSize = 16;
-        g.setFont(new Font("Helvetica", Font.PLAIN, fontSize));
+        g.setFont(new Font(MHQConstants.FONT_HELVETICA, Font.PLAIN, fontSize));
```

Comment:
Helvetica is typically not present on Win systems, so this is a bit of a doubtful font choice. But not a problem of this PR.